BlockServer/core/config_list_manager.py

```diff
@@ -358,3 +360,5 @@ def update_monitors(self):
             self._bs.setParam(BlockserverPVNames.COMPS, compress_and_hex(convert_to_json(self.get_components())))
             # Update them
             self._bs.updatePVs()
+
+        AlarmConfigLoader.get_instance().load_in_new_thread()
```

Comment:
What if two threads call this simultaneously? It could lead to multiple copies of the batch script running the same set of commands with an offset which could be bad. Could we check whether process.kill() in the load actually kills off the existing script? A pause in the batch script would make it easier to trigger this.